### PR TITLE
Use instance name

### PR DESCRIPTION
**Pull request by maggarwal13**

### CODE REVIEW

File: whois-commons/src/main/java/net/ripe/db/whois/common/WhoisRipePropertyResolver.java [hunk 1]
```diff
@@ -16,6 +22,28 @@
...
+        String hostName = System.getenv("HOSTNAME");
+        if (StringUtils.isBlank(hostName)) {
+            try {
+                hostName = InetAddress.getLocalHost().getHostName();
+            } catch (UnknownHostException ignored) {
+                LOGGER.debug("{}: {}", ignored.getClass().getName(), ignored.getMessage());
+            }
+
+            if (StringUtils.isBlank(hostName)) {
+                System.err.println("Acquiring HOSTNAME failed! Try\nexport HOSTNAME=$(hostname -s)\n");
```
Comment (eshryane) [inline whois-commons/src/main/java/net/ripe/db/whois/common/WhoisRipePropertyResolver.java:37]: Use `Logger.error() `not `System.err.println()`

File: whois-query/src/test/java/net/ripe/db/whois/query/pipeline/ConnectionPerIpLimitHandlerTest.java [hunk 2]
```diff
@@ -46,6 +46,7 @@
 
     @Before
     public void setUp() {
+        System.setProperty("instance.name", "10.0.0.0");
```
Comment (eshryane) [inline whois-query/src/test/java/net/ripe/db/whois/query/pipeline/ConnectionPerIpLimitHandlerTest.java:49]: Make sure to use `System.clearProperty("instance.name")` in an `@After` method also.

File: whois-rpsl/src/main/java/net/ripe/db/whois/common/domain/Hosts.java [hunk 3]
```diff
@@ -1,35 +1,19 @@
...
-            }
-
-            if (StringUtils.isBlank(hostName)) {
-                System.err.println("Acquiring HOSTNAME failed! Try\nexport HOSTNAME=$(hostname -s)\n");
-                System.exit(1);
-            }
+        instanceName = System.getProperty("instance.name");
+        if (StringUtils.isBlank(instanceName)) {
+            System.err.println("Instance name is not defined");
+            System.exit(1);
```
Comment (eshryane) [inline whois-rpsl/src/main/java/net/ripe/db/whois/common/domain/Hosts.java:12]: Don't use `System.exit()` but throw an `IllegalStateException`

File: whois-update/src/test/java/net/ripe/db/whois/update/handler/response/ResponseFactoryTest.java [hunk 4]
```diff
@@ -62,6 +62,7 @@
 
     @Before
     public void setUp() throws Exception {
+        System.setProperty("instance.name", "10.0.0.0");
```
Comment (eshryane) [inline whois-update/src/test/java/net/ripe/db/whois/update/handler/response/ResponseFactoryTest.java:65]: Add `System.clearProperty`

File: whois-commons/src/main/java/net/ripe/db/whois/common/WhoisRipePropertyResolver.java (hunk 1 above)
Comment (maggarwal13) [reply whois-commons/src/main/java/net/ripe/db/whois/common/WhoisRipePropertyResolver.java:37]: Yes, I just copy pasted the old the code. I will correct this one

**Review by eshryane — APPROVED** [review]

**Pull request closed without merging**